gnomad_qc/v5/resources/basics.py

```diff
@@ -1,14 +1,52 @@
 """Script containing generic resources."""
 
-from gnomad.resources.resource_utils import VariantDatasetResource
+import logging
+from typing import List, Optional, Set, Union
 
-from gnomad_qc.v5.resources.constants import CURRENT_VERSION, WORKSPACE_BUCKET
+import hail as hl
+from gnomad.resources.resource_utils import (
+    VariantDatasetResource,
+    VersionedVariantDatasetResource,
+)
+from hail.utils import new_temp_file
+
+from gnomad_qc.v4.resources.basics import _split_and_filter_variant_data_for_loading
+from gnomad_qc.v5.resources.constants import (
+    AOU_BAD_QUALITY_PATH,
```

Comment:
nitpick -- could you rename this to `AOU_LOW_QUALITY_PATH`, `AOU_REMOVE_PATH`, or something similar?